unit/goto-programs/goto_program_instruction.cpp

```diff
@@ -0,0 +1,110 @@
+/*******************************************************************\
+
+  Module: Unit tests for goto_program::validate
```

Comment:
Nit pick: no space before Module, Author

Reply:
Done.

Reply:
Needs to be addressed again.